angular/spec/test.ts

```diff
@@ -0,0 +1,28 @@
+import { webcrypto } from "crypto";
+import "jest-preset-angular/setup-jest";
+
+Object.defineProperty(window, "CSS", { value: null });
```

Comment:
Are these polyfills to get the JS happy running in `jsdom`?

Reply:
yes, same with the `webcrypto` stuff